apps/authentication/serializers/face.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+__all__ = [
+    'FaceCallbackSerializer'
+]
+
+
+class FaceCallbackSerializer(serializers.Serializer):
+    token = serializers.CharField(required=True, allow_blank=False)
+    success = serializers.BooleanField(required=True, allow_null=False)
+    error_message = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+    face_code = serializers.CharField(required=False, allow_null=True, allow_blank=True)
+
+    def update(self, instance, validated_data):
+        pass
+
+    def create(self, validated_data):
+        pass
```

Comment:
There don't seem to be any major issues with the provided code snippet, but I might suggest some minor improvements in the comment section:

# Add a module docstring

    from rest_framework.serializers import (  # This is an updated recommendation 
...,

Ensure that the class name `FaceCallbackSerializer` matches its intended use, making it clear what this serializer does.

Additionally, you can replace the commented-out lines `[create], [update]` with appropriate method implementations inside each class.
This will provide clearer functionality and maintain code readability.